resources/problem-vote.scss

```diff
@@ -0,0 +1,54 @@
+.problem-vote-form {
+    margin: 1em;
+    min-width: 25em;
+}
+
+.problem-vote-form-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: flex-end;
+    border-bottom: 1px solid #aaa;
+}
+
+.problem-vote-form-title {
+    font-size: 2em;
+    margin-right: 0.5em;
+}
+
+.problem-vote-date {
+    font-style: italic;
+}
+
+.problem-vote-form-info {
+    font-size: 1.2em;
+    margin-right: 0.5em;
+}
+
+#problem-vote-form textarea {
```

Comment:
Is there a reason we have both an id and a class for `problem-vote-form`?

Reply:
Lack of imagination. It's probably ok to rename:
* `class="problem-vote-form"` -> `class="problem-vote-container"`
* `.problem-vote-form` -> `.problem-vote-container`